gtsam/nonlinear/NonlinearEquality.h

```diff
@@ -87,7 +87,8 @@ class NonlinearEquality: public NoiseModelFactor1<VALUE> {
    * Constructor - forces exact evaluation
    */
   NonlinearEquality(Key j, const T& feasible,
-      const CompareFunction &_compare = boost::bind(traits<T>::Equals,_1,_2,1e-9)) :
+      const CompareFunction &_compare = boost::bind(traits<T>::Equals,
```

Comment:
Format just the edited sections? This is pretty easy with VS code.

Reply:
Sure